### PR TITLE
Add requestPrincipal with issuer

### PR DESCRIPTION
**Pull request by werdes72**
See: #313

### CODE REVIEW

File: internal/builders/istio.go [hunk 1]
```diff
@@ -139,10 +140,25 @@ func (rf *FromBuilder) Get() *v1beta1.Rule_From {
...
 	// Only support one source at the moment
-	source := v1beta1.Source{RequestPrincipals: []string{"*"}}
+	var requestPrincipals []string
+	for _, strategy := range accessStrategies {
+		if strategy.Name == "jwt" {
+			authentications := &Authentications{
+				Authentications: []*Authentication{},
+			}
+			if strategy.Config != nil {
+				_ = json.Unmarshal(strategy.Config.Raw, authentications)
```
Comment (videlov) [inline internal/builders/istio.go:153]: please do not ignore errors when unmarshalling Raw content. did you test if there is `Authorizations` in config as well? if not please add a unit test for it.

**Review by videlov — CHANGES_REQUESTED** [review]

**Comment by videlov** [comment]
please also go over DoDs for the task, e.g. docs and integration test

File: internal/builders/istio.go (hunk 1 above)
Comment (werdes72) [reply internal/builders/istio.go:153]: It's the same way of getting that issuer as in:
https://github.com/kyma-project/api-gateway/blob/main/internal/builders/istio.go#L332
We're not checking for error there either.

Added `Authorizations` checking in unit test.

Comment (videlov) [reply internal/builders/istio.go:153]: we should fix both then? this is hiding a potential error, which relies solely on validation and we can not be certain all cases are checked. having error in such places will reduce time when investigating issues.

Comment (videlov) [reply internal/builders/istio.go:153]: seems like `json.Umarshal` on validate is enough to caught issues on Raw config level.

**Review by videlov — APPROVED** [review]

**Pull request merged**